src/js/nav/buildNav.js

```diff
@@ -0,0 +1,74 @@
+const yaml = require('js-yaml');
+const axios = require('axios');
+const _ = require('lodash');
+const fs = require('fs');
+
+axios.get('https://raw.githubusercontent.com/RedHatInsights/cloud-services-config/enhancements/chrome-nav/main.yml')
+.then(response => yaml.safeLoad(response.data)).then(data => buildNavFromConfig(data));
+
+function buildNavFromConfig(masterConfig) {
```

Comment:
This would be better to dispatch new action to redux and store current nav based on the app instead of storing it in JSON file.